dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -35,23 +42,18 @@
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
-
+import org.eclipse.aether.RepositoryException;
```

Comment:
we need to standardize the import order; there's a bit of a discrepancy between the external and internal stylesheets